datus/tools/func_tool/sub_agent_task_tool.py

```
@@ -0,0 +1,523 @@
+
+            return ExtKnowledgeNodeInput(user_message=prompt)
+
+        # Generic fallback for other agentic node types
+        from datus.schemas.base import BaseInput
+
+        # Try to use the node's type-specific input if available
+        try:
+            input_cls = NodeType.type_input(node.type, {}, ignore_require_check=True)
+            if hasattr(input_cls, "user_message"):
+                input_cls.user_message = prompt
+            return input_cls
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)